website/content/en/docs/cloud-providers/AWS/launch-templates.md

```diff
@@ -0,0 +1,236 @@
+4. Update the Provisioner CRD to specify the new Launch Template.
+
+An example yaml cloudformation definition of a launch template for Karpenter is
+provided below. 
+
+Cloudformation yaml is suited for the moderately high configuration density of
+launch templates, and creating the unusual InstanceProfile resource. 
+
+You must manually replace these values in the template:
+- SecurityGroupID
+  - list all security groups with `aws ec2 describe-security-groups`
+- Parameters in UserData
+
+```yaml
+AWSTemplateFormatVersion: '2010-09-09'
+Resources:
+  # create InstanceProfile wrapper on NodeRole
+  KarpenterNodeInstanceProfile:
+    Type: "AWS::IAM::InstanceProfile"
+    Properties:
+      InstanceProfileName: "KarpenterNodeInstanceProfile"
+      Path: "/"
+      Roles:
+        - Ref: "KarpenterNodeRole"
+  # create role with basic permissions for EKS node
+  KarpenterNodeRole:
+    Type: "AWS::IAM::Role"
+    Properties:
+      RoleName: "KarpenterNodeRole"
+      Path: /
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                !Sub "ec2.${AWS::URLSuffix}"
+            Action:
+              - "sts:AssumeRole"
+      ManagedPolicyArns:
+        - !Sub "arn:${AWS::Partition}:iam::aws:policy/AmazonEKSWorkerNodePolicy"
+        - !Sub "arn:${AWS::Partition}:iam::aws:policy/AmazonEKS_CNI_Policy"
+        - !Sub "arn:${AWS::Partition}:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly"
+        - !Sub "arn:${AWS::Partition}:iam::aws:policy/AmazonSSMManagedInstanceCore"
+  MyLaunchTemplate:
+    Type: AWS::EC2::LaunchTemplate
+    Properties: 
+      LaunchTemplateData: 
+        IamInstanceProfile:
+          # Get ARN of InstanceProfile defined above
+          Arn: !GetAtt
+            - KarpenterInstanceProfile
+            - Arn
+        ImageId: ami-074cce78125f09d61
+        # UserData is Base64 Encoded
+        UserData: !Base64 > 
+            #!/bin/bash
+            /etc/eks/bootstrap.sh <my-cluster-name> \
+            --kubelet-extra-args <'--max-pods=40'> \
+            --b64-cluster-ca <certificateAuthority> \
+            --apiserver-endpoint <endpoint> 
+            --dns-cluster-ip <serviceIpv4Cidr>
+            --use-max-pods false
+        BlockDeviceMappings: 
+          - Ebs:
+              VolumeSize: 80
+              VolumeType: gp3
+            DeviceName: /dev/xvda
+        # The SecurityGroup must be associated with the cluster VPC
+        SecurityGroupIds:
+          - sg-a69adfdb
+      LaunchTemplateName: KarpenterCustomLaunchTemplate
+```
+
+Create the Launch Template by uploading the CloudFormation yaml file. The
+sample yaml creates an IAM Object (InstanceProfile), so `--capabilities
+CAPABILITY_NAMED_IAM` must be indicated.
+
+```
+aws cloudformation create-stack \
+  --stack-name KarpenterLaunchTemplateStack \
+  --template-body file:///Users/gcline/Desktop/lt-cfn-demo.yaml \

```

Comment:
can you remove your `gcline` path? :D  

Maybe just replace with `file://$(pwd)/lt-cfn-demo.yaml`